imgui.go

```diff
@@ -982,16 +990,16 @@ func IsMouseDoubleClicked(button int) bool {
 	return C.iggIsMouseDoubleClicked(C.int(button)) != 0
 }
 
-// Columns calls ColumnsV(count, label, ColumnsFlagsNone).
+// Columns calls ColumnsV(count, label, false).
 func Columns(count int, label string) {
-	ColumnsV(count, label, ColumnsFlagsNone)
+	ColumnsV(count, label, false)
 }
 
 // ColumnsV creates a column layout of the specified number of columns.
-func ColumnsV(count int, label string, flags int32) {
+func ColumnsV(count int, label string, border bool) {
```

Comment:
This signature change (together with the deletion of the ColumnFlags) warrants a switch to v2.0.0 for the wrapper as it is an incompatible change.
It seems like this is inevitable. It may be a good reason for me to look into how to properly handle major releases with Go.